doc/sphinx/Pacemaker_Explained/collective.rst

```diff
@@ -348,9 +349,11 @@ _____________
    |                   |                 |    single: option; promoted-node-max (clone)          |
    |                   |                 |    single: promoted-node-max; clone option            |
    |                   |                 |                                                       |
-   |                   |                 | If ``promotable`` is **true** and ``globally-unique`` |
-   |                   |                 | is **false**, the number of clone instances can be    |
-   |                   |                 | promoted at one time on a single node                 |
+   |                   |                 | If ``promotable`` is **true**, ``globally-unique``    |
+   |                   |                 | is not **false**, and ``clone-node-max`` is greater   |
```

Comment:
Previously it was `is **false**`. That looks like it was a typo before, but I want to confirm.

Also, I don't know that we want to make this any more verbose, but I think we're really talking about when it's not EXPLICITLY false. Say it defaults to false because `clone-node-max = 1`. Then I presume `promoted-node-max = 0` would still have an effect.